### PR TITLE
Create footer

### PR DESCRIPTION
**Pull request by andrewkdinh**
Replicated this mockup footer

footer

### CODE REVIEW

File: partials/footer.hbs [hunk 1]
```diff
@@ -2,28 +2,57 @@
...
+        </form>
+      </div>
+    </div>
+    <nav class="footer-right">
+      <div class="footer-col">
+        <ul>
+          <li><a href="." rel="noopener">Home</a></li>
+          <li><a href="about" rel="noopener">About</a></li>
+          <li><a href="events" rel="noopener">Events</a></li>
+          <li><a href="apply" rel="noopener">Apply</a></li>
```
Comment (PotatoParser) [inline partials/footer.hbs]: Switch href to absolute path: `{{ @site.url }}/<route to your link>`

File: partials/footer.hbs [hunk 2]
```diff
@@ -2,28 +2,57 @@
...
+        </ul>
+      </div>
+      <div class="footer-col">
+        <ul>
+          <li><a href="micromouse" rel="noopener">Micromouse</a></li>
+          <li><a href="hope" rel="noopener">HOPE</a></li>
+          <li><a href="industry" rel="noopener">Industrial Relations</a></li>
```
Comment (PotatoParser) [inline partials/footer.hbs]: Switch href to absolute path: `{{ @site.url }}/<route to your link>`

File: src/sass/components/_footer.scss [hunk 3]
```diff
@@ -1,67 +1,105 @@
...
-      display: inline;
-    }
+.footer-social a:last-of-type {
+  margin-right: 0px;
+}
 
-    &:nth-child(2) {
-      display: none;
+.footer-mailing-list button {
+  background-color: transparent;
```
Comment (PotatoParser) [inline src/sass/components/_footer.scss:52]: Add `border-style: solid;` instead of using the default of `outset`

File: partials/footer.hbs [hunk 4]
```diff
@@ -2,28 +2,57 @@
...
+        </ul>
+      </div>
     </nav>
-    <p class="m-footer-copyright jslicense">
-      <a href="{{asset "html/javascript.html"}}" rel="jslicense">{{t "JavaScript license information"}}</a>
-    </p>
+  </div>
+  <div class="footer-copyright">
+    <p>{{@site.title}} &copy; {{date format="YYYY"}}</p>
```
Comment (PotatoParser) [inline partials/footer.hbs]: I'm not sure if this is the correct copyright that we want to see. I think the site title is different from our actual copyright. `© Copyright 2020 IEEE Student Branch at UC Berkeley`

File: src/sass/components/_footer.scss [hunk 5]
```diff
@@ -1,67 +1,105 @@
...
+
+.footer-col li {
+  margin: 10px 0px;
+}
+
+.footer-col a {
+  display: block;
+  font-size: 18px;
+  font-weight: bolder;
+  color: rgba(253, 180, 21);
```
Comment (PotatoParser) [inline src/sass/components/_footer.scss]: This color is a little warm, the hexcode for the color in the image is `#fed966`

File: src/sass/components/_footer.scss [hunk 6]
```diff
@@ -1,67 +1,105 @@
...
+  margin: 0px;
+  padding-left: 0px;
+  list-style-type: none;
+}
+
+.footer-col li {
+  margin: 10px 0px;
+}
+
+.footer-col a {
```
Comment (PotatoParser) [inline src/sass/components/_footer.scss:91]: Needed hover animation (to help users see what they are hovering over)

File: src/sass/components/_footer.scss [hunk 7]
```diff
@@ -1,67 +1,105 @@
...
-    @include respond-to('medium') {
-      display: inline;
-    }
+.footer-social a:last-of-type {
+  margin-right: 0px;
+}
 
-    &:nth-child(2) {
-      display: none;
+.footer-mailing-list button {
```
Comment (PotatoParser) [inline src/sass/components/_footer.scss:51]: There seems to be a small border-radius on the image, perhaps adding `border-radius: 2px;` would work.

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Overall a very clean job! There are just a few visual elements that are off.

**Review by encadyma — APPROVED** [review]
Any reason why the mailing list border is outsetted? Otherwise, it looks good.

**Review by sulattphone — APPROVED** [review]
Looks solid.
I wonder how it would look if we have our IEEE logo right in the middle of the footer? Can we see that?

**Comment by andrewkdinh** [comment]
> Any reason why the mailing list border is outsetted? Otherwise, it looks good.

What do you mean it’s outsetted?

**Comment by PotatoParser** [comment]
@andrewkdinh Don't worry! You resolved the issue in your latest commit!

**Comment by andrewkdinh** [comment]
@sulattphone I added the IEEE logo to the footer but made it disappear for smaller devices because it felt too cluttered

Screen Shot 2020-05-19 at 8 33 00 PM

**Comment by sulattphone** [comment]
> @sulattphone I added the IEEE logo to the footer but made it disappear for smaller devices because it felt too cluttered
> 
> Screen Shot 2020-05-19 at 8 33 00 PM

Oh I love this actually. Yeah I agree that it would be too much for the mobile screens. Alright we can discuss more about all these designs at the meeting.

**Pull request closed without merging**